draft-authors-cdrplus-infosec-baseline.md

```diff
@@ -113,6 +113,8 @@ In addition, the authorisation server:
 1. **MUST** support the [@!OIDC-Core] scopes `openid` and `profile`;
 2. **SHALL** support signed ID Tokens and **MAY** support signed and encrypted ID Tokens;
 1. **SHALL** issue access tokens with an expiry time of between 2 and 10 minutes;
+1. **SHALL NOT** support public clients
```

Comment:
FAPI 1.0 Advanced states in the introduction:
> This profile does not support public clients.

I don't think it is worthwhile restating this.

Reply:
Ahhhhh. My mistake. Baseline allows them but advanced doesn't. 

Yeah agreed doesn't need restating. That was pebcak sorry.